src/Surfnet/StepupMiddleware/MiddlewareBundle/Console/Command/ReplaySpecificEventsCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddleware\MiddlewareBundle\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class ReplaySpecificEventsCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('middleware:event:specific_replay')
+            ->setDescription('replay specified events for specified projectors');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        /** @var FormatterHelper $formatter */
+        $formatter = $this->getHelper('formatter');
+
+        /** @var QuestionHelper $interrogator */
+        $interrogator = $this->getHelper('question');
+
+        $output->writeln(['', $formatter->formatBlock([
+            '',
+            'WARNING!!!!',
+            '',
+            'You are about to WIPE all read data and recreate all data based on chosen events and projectors.',
```

Comment:
This should plain simply not be true. As discussed yesterday - this is intended to be a proper event replay, through specific replay-projectors.

In general: please remove the copy pasted code. For now just a message that it does not actually do anything should suffice :wink: